### ISSUE TITLE
Sphinx doc build does not support superscript or substitution (Trac #1164)

### ISSUE THREAD

**Issue by butlerpd** [SasView/sasview#1203]
While fixing a syntax issue with the superscript on `slit_calculator_help.rst` I discovered that even properly formatted superscripts such as: 

```text
xxx\ :sup:`a`
```
do not render. Moreover 1\|Ang| does not substitute |Ang| despite it being listed in `sasview/docs/sphinx-docs/source/prolog.rst` Further testing by including a replace directive in the same help.rst file did not work either.  At least getting superscripts working would be nice.

Migrated from http://trac.sasview.org/ticket/1164
```json
{
    "status": "closed",
    "changetime": "2018-09-06T07:58:39",
    "_ts": "2018-09-06 07:58:39.214745+00:00",
    "description": "While fixing a syntax issue with the superscript on `slit_calculator_help.rst` I discovered that even properly formatted superscripts such as: \n{{{\nxxx\\ :sup:`a`\n}}}\ndo not render. Moreover 1\\|Ang| does not substitute |Ang| despite it being listed in `sasview/docs/sphinx-docs/source/prolog.rst` Further testing by including a replace directive in the same help.rst file did not work either.  At least getting superscripts working would be nice.",
    "reporter": "butler",
    "cc": "",
    "resolution": "fixed",
    "workpackage": "SasView Framework Enhancements",
    "time": "2018-08-25T02:06:43",
    "component": "SasView",
    "summary": "Sphinx doc build does not support superscript or substitution",
    "priority": "major",
    "keywords": "",
    "milestone": "SasView 4.2.0",
    "owner": "",
    "type": "defect"
}
```

**Comment by smk78** [comment]
Trac update at `2018/09/06 07:49:34`: **smk78** commented:

In PR 168 @pkienzle said:

Fix the problem with sphinx superscript and substitution (Ticket 1164) in an emphasized environment by removing the emphasis.

Not the best solution. May want to try the most recent version of sphinx under python 3 to see if this problem has already been fixed upstream.

Other option is to turn off emphasis for the superscript and the substitution.


@butlerpd replied:

Need to check output yet but a couple of comments:

If Prolog.rst is not used perhaps we should remove.

Removing the emphasis should be fine but since it already says "Note!" maybe making it a note . i.e.
:: Note: xyz would be appropriate?


@butlerpd then added:

Reading this again I don't believe using the NOTE markup is most appropriate -- this fix is fine for this and given the number of times this seems to happen, waiting for 5.0 and the upgrade to python 3.0 etc seems to be reasonable.

**Comment by smk78** [comment]
Trac update at `2018/09/06 07:51:39`: **smk78** commented:

Leave prolog.rst alone for the time being due to possible unpredictable consequences on other rst docs.

**Comment by smk78** [comment]
Trac update at `2018/09/06 07:57:32`: **smk78** commented:

Removing emphasis fine for now. Closes #1203

**Comment by smk78** [comment]
Trac update at `2018/09/06 07:58:39`: 
* **smk78** changed resolution from "" to "fixed"
* **smk78** changed status from "new" to "closed"
